### DISCUSSION TITLE
How do i install talsec for react native older version . Application version is 0.62.0

### DISCUSSION THREAD

**Discussion by bharath-s-caritorsolutions** [talsec/Free-RASP-ReactNative#55] [Q&A]
Support needed for older version . 

Application details. 
React native : 0.62.0

**Comment by tompsota** [comment]
Hi @bharath-s-caritorsolutions,

I tested freeRASP with your version of RN and found a compatibility problem in the Android implementation. It will be resolved in the next release. However, if you are referring to some specific error, please, provide the stacktrace as we might have missed something. 

Additionally, I would strongly suggest to update to a newer RN version. We are always aiming to support the latest versions of RN, ensuring smooth integration of freeRASP.

Cheers,
Tomas from Talsec

**Comment by tompsota** [comment]
Hello, @bharath-s-caritorsolutions,

today we released freeRASP@3.7.1. 

Can you try to integrate this version to see if it fixes your issue?

Thanks,

Tomas from Talsec
